### DISCUSSION TITLE
[速报] 爱发电域名被停止解析，ICP 备案号已注销

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#4829] [综合讨论 / General]
afdian.net 被 clientHold，其 ICP 备案号经查询已注销，公安备案仍存在

目前服务器没关，所以还可以通过修改 host 来解析访问：`154.8.151.42        afdian.net`

@the1812

**Comment by Tinhone** [comment]
![image](https://github.com/user-attachments/assets/1a04685f-82df-49ce-a919-5df038b3245b)

**Comment by the1812** [comment]
我咋还能正常访问

**Comment by Alexius66** [comment]
我也能正常访问，无论是否使用代理

**Reply by Alexius66** [reply]
> 我也能正常访问，无论是否使用代理

更新：7月16日16:00，已经无法正常访问

**Reply by bilibilijandy** [reply]
> > 我也能正常访问，无论是否使用代理
> 
> 更新：7月16日16:00，已经无法正常访问

+1

**Comment by bilibilijandy** [comment]
等吧，过几天应该就好
